### PR TITLE
Add CircleCI badge to README

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

**Comment by Etiene** [comment]
Screen Shot 2019-03-20 at 13 32 29
Any ideas why it's not showing up properly when I click on view file?

**Comment by gliptak** [comment]
@Etiene you are correct. Currently it only shows up when I'm logged in into CircleCI. There is probably a setting on the project to allow anonymous read.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Terratest
 
 [![Maintained by Gruntwork.io](https://img.shields.io/badge/maintained%20by-gruntwork.io-%235849a6.svg)](https://gruntwork.io/?ref=repo_terratest)
+[![CircleCI](https://circleci.com/gh/gruntwork-io/terratest.svg?style=svg)](https://circleci.com/gh/gruntwork-io/terratest)
```
Comment (Etiene) [inline README.md]:
```suggestion
[![CircleCI](https://circleci.com/gh/gruntwork-io/terratest.svg?style=svg&circle-token=e48019e09fc3b8bf6e0315a84048501c87c4157c)](https://circleci.com/gh/gruntwork-io/terratest)
```

Comment (gliptak) [reply README.md]: @Etiene thank you. Even with this above, https://circleci.com/gh/gruntwork-io/terratest still doesn't have anonymous access ...

**Pull request merged**